.vib/prestashop/goss/goss.yaml

```diff
@@ -0,0 +1,32 @@
+command:
+  apcu-caching-support:
+    exec: /opt/bitnami/php/bin/php -i | grep --quiet apcu
+    exit-status: 0
+    stdout: [ ]
+    stderr: [ ]
+    timeout: 40000
+  prestashop-version-ok:
+    exec: cat /opt/bitnami/prestashop/app/AppKernel.php | grep --quiet $(echo $APP_VERSION | tr "-" ".")
+    exit-status: 0
+    stdout: [ ]
+    stderr: [ ]
+    timeout: 40000
+  prestashop-cli-install-module:
+    exec: /opt/bitnami/php/bin/php /opt/bitnami/prestashop/bin/console prestashop:module install ps_categoryproducts
```

Comment:
Is this module something external that might be removed in the future?
I'd take the same attitude for this as we do for plugins: try not to include them in our tests since they tend to come and go. WDYT?

Reply:
The module I was installing is developed by Prestashop itself, so there are fewer changes for it to be deprecated out of the blue.

Nevertheless, I agree with your point and have replaced the test with something less prone to change.